### ISSUE TITLE
Option to change audio quality

### ISSUE THREAD

**Issue by notramo** [vector-im/riot-web#12739]
Users with good microphones, fast computers, and fast network may want to have higher quality audio.
The Opus codec supports 6 kb/s to 510 kb/s bitrate! Why not utilize it?
High quality audio can be beneficial in many scenarios:
- where high dynamic range is required, e.g. lot of people, some are further from the microphone (quiet), some are closer to it (louder)
- musical events, music education
- good quality microphone and headphones - Why not enjoy utilizing it fully to have a good conversation?

On the other hand, on slow networks, where the bitrate is too much, some people would settle for the lowest quality just to be able to talk.

I suggest that users could specify the bitrate with digits would be the most flexible solution. It should be changeable during a call, and Riot should save the last used bitrate for each user.

**Comment by marc107** [comment]
Appreciate this!

**Comment by jryans** [comment]
Since Riot uses Jitsi for calls, it sounds like this is most likely to happen if a request is filed with https://github.com/jitsi/jitsi-meet/issues.

**Comment by notramo** [comment]
@jryans Does Riot use Jitsi for 1:1 calls?

**Comment by jryans** [comment]
> @jryans Does Riot use Jitsi for 1:1 calls?

No. For 1:1 calls, Riot uses P2P WebRTC directly. It would depend on whether browsers offer such an option to control audio quality in that case.

**Comment by jryans** [comment]
If this is more important for group vs. 1:1 calls, it might make sense to track with separate issues, as they are different paths, each with their own challenges.

**Comment by BloodyIron** [comment]
Yeah really wish we could have actual controls over codecs and bitrates in 1:1. The quality really is not as good as I was hoping for. And getting 64-128kbps of upload rate is trivial now adays. We need options for those on limited connections, and those with plenty of bandwidth.

I too am really interested in this.
